# Update CMakeLists.txt to build for KDE 4.2.

Index: kio-ftps-0.2/CMakeLists.txt
===================================================================
--- kio-ftps-0.2.orig/CMakeLists.txt	2009-04-19 03:01:49.000000000 +0200
+++ kio-ftps-0.2/CMakeLists.txt	2009-04-20 01:53:06.000000000 +0200
@@ -8,7 +8,7 @@
 
 kde4_add_plugin( kio_ftps ${mySources} )
 
-target_link_libraries(kio_ftps ${KDE4_KDECORE_LIBS} kio )
+target_link_libraries(kio_ftps ${KDE4_KDECORE_LIBS} ${KDE4_KIO_LIBS} )
 
 install(TARGETS kio_ftps  DESTINATION ${PLUGIN_INSTALL_DIR} )
 install( FILES ftps.protocol  DESTINATION  ${SERVICES_INSTALL_DIR} )
