# Update CMakeLists.txt to build for KDE 4.2.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -8,7 +8,7 @@ set(mySources ftp.cpp)
 
 kde4_add_plugin( kio_ftps ${mySources} )
 
-target_link_libraries(kio_ftps ${KDE4_KDECORE_LIBS} kio )
+target_link_libraries(kio_ftps ${KDE4_KDECORE_LIBS} ${KDE4_KIO_LIBS} )
 
 install(TARGETS kio_ftps  DESTINATION ${PLUGIN_INSTALL_DIR} )
 install( FILES ftps.protocol  DESTINATION  ${SERVICES_INSTALL_DIR} )
